### PR TITLE
Update workspace's name

### PR DESCRIPTION
**Pull request by lenville**

### CODE REVIEW

File: keymaps/csscomb.cson [hunk 1]
```diff
@@ -7,5 +7,5 @@
 
 # For more detailed documentation see
 # https://atom.io/docs/latest/advanced/keymaps
-'.workspace':
+'.atom-workspace':
```
Comment (stramel) [inline keymaps/csscomb.cson]: I believe this should be `atom-workspace` instead of `.atom-workspace`. It says `tag` instead of `class`.

**Comment by lenville** [comment]
Sorry, I didn't read it carefully. But it really does work at all :sweat_smile:

**Comment by stramel** [comment]
All good, once that changes you got my approval! :+1:

**Comment by stramel** [comment]
This PR should be closed in favor of #18 which does the same thing and is already using the correct element selector.

**Comment by stramel** [comment]
Closing in favor of #18

**Pull request closed without merging**